src/core/FBXWrangler.cpp

```diff
@@ -5101,6 +5106,20 @@ void FBXWrangler::buildCollisions()
 		NiAVObjectRef ni_parent = DynamicCast<NiAVObject>(conversion_Map[rb->GetParent()]);
 		ni_parent->SetCollisionObject(build_physics(rb, meshes));
 	}
+	for (const auto& rb : physic_entities)
```

Comment:
Previously constraints were being built during building of body, so if target body wasn't built at this point, it crashed.